docker/Dockerfile-ubuntu-20

```diff
@@ -40,7 +40,7 @@ COPY ./CMakeLists.txt ./CMakeLists.txt
 
 WORKDIR /app/build
 
-RUN cmake .. -G Ninja -DCMAKE_BUILD_TYPE=Release -DDEV_MODE=ON && \
+RUN cmake .. -G Ninja -DCMAKE_BUILD_TYPE=Release -DDEV_MODE=ON  -DSANITIZE_ADDRESS=ON && \
```

Comment:
Is it possible to set it as external parameter? I don't think we should set it as default
`docker build -t [image-name] --build-arg [arg-variable]=[value] .`
https://phoenixnap.com/kb/docker-environment-variables
And just extend readme for test build like:
to build in docker with different sanitizers use this command...